src/optimizer/acc_hash_join_optimizer.cpp

```diff
@@ -264,7 +264,7 @@ static bool isBuildSideQualified(LogicalOperator* buildRoot) {
     while (op->getNumChildren() == 1) {
         op = op->getChild(0).get();
     }
-    return op->getOperatorType() == LogicalOperatorType::GDS_CALL;
+    return op->getOperatorType() == LogicalOperatorType::RECURSIVE_EXTEND;
```

Comment:
Was this a "bug" left from previous PR? I don't see why GDS_CALL now becomes RECURSIVE_EXTEND.

Reply:
Yeah this was a bug left in previous refactor